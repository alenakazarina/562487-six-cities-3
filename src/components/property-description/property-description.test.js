import React from 'react';
import renderer from 'react-test-renderer';
import PropertyDescription from './property-description';

const description = `Property description`;

describe(`PropertyDescription`, () => {
  it(`should render PropertyDescription correctly`, () => {
    const tree = renderer.create(
        <PropertyDescription description={description} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
