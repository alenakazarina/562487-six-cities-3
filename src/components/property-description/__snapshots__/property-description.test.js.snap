// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyDescription should render PropertyDescription correctly 1`] = `
<div
  className="property__description"
>
  <p
    className="property__text"
  >
    Property description
  </p>
</div>
`;
